installers/olm/README.md

```diff
@@ -0,0 +1,6 @@
+1. Set VERSION env variable
+2. Update rbac and operator in config
+2. Choose the mode (cluster or namespace) and update bundle/kustomization.yaml and makefile with necessary mode.
+3. update description in bundle.csv.yaml
+4. Update variables in makefile.
+5. ```make bundles```
```

Comment:
When I run `make` from correct folder I get `gawk` error (after some files were already generated). Maybe we should check presence of all dependencies before starting the generation?
```
Generating kustomize files in config/manifests
WARN[0000] Skipping definitions parsing: APIs root dir "api" does not exist 
Kustomize files generated successfully
./generate.sh: line 141: gawk: command not found
make: *** [bundles/community] Error 127
```

Reply:
fixed